packages/ckeditor5-ui/tests/colorpicker/colorpickerview.js

```diff
@@ -50,31 +50,179 @@ describe( 'ColorPickerView', () => {
 
 			clock.tick( 200 );
 
-			expect( view.input.fieldView.value ).to.equal( '#0000ff' );
+			expect( view.inputRow.children.get( 1 ).fieldView.value ).to.equal( '0000ff' );
 		} );
 
 		it( 'should update color property after getting an input', () => {
-			view.input.fieldView.value = '#ff0000';
-			view.input.fieldView.fire( 'input' );
+			view.inputRow.children.get( 1 ).fieldView.value = '#ff0000';
+			view.inputRow.children.get( 1 ).fieldView.fire( 'input' );
 
 			clock.tick( 200 );
 
 			expect( view.color ).to.equal( '#ff0000' );
 		} );
 
-		// Requires https://github.com/ckeditor/ckeditor5/pull/13819 - text input should be properly registered as a focusable.
-		it( 'should not update color property during input editing when focused', () => {
-			view.color = '#000000';
+		describe( 'should update color property', () => {
```

Comment:
I don't see test cases that ensures the input value gets trimmed.

As far as I saw it's working fine but we need to have it covered.